src/components/stock-popup.js

```diff
@@ -67,7 +67,7 @@ class StockPopUp {
 
     // retrieve watchlist array from storage
     let watchlist = store.get('watchlist') || []; // ['AAPL', 'AMD'] or []
-    let hasStock = watchlist.includes(this.companyId); // true/false
+    let hasStock = watchlist.includes(`${this.companyId} | ${this.companyName}`); // true/false
```

Comment:
Register this in the constructor so you don't have to keep registering it multiple times in different functions.